issuetracker/internal/azureboards/azureboards_test.go

```diff
@@ -0,0 +1,33 @@
+package azureboards
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_IssueTracker_IssueURLFor(t *testing.T) {
+
+	var tests = []struct {
+		input  string
+		taskID string
+		want   string
+	}{
+		{"https://dev.azure.com/todoerruser/todocheck", "1", "https://dev.azure.com/todoerruser/todocheck/_apis/wit/workitems/1?api-version=6.0"},
```

Comment:
You can add name for every test in order to increase readability and to be clear what is the purpose of every test.